BitwardenShared/Core/Auth/Services/AuthService.swift

```diff
@@ -207,6 +207,7 @@ protocol AuthService {
     func requirePasswordChange(
         email: String,
         masterPassword: String,
+        preAuth: Bool,
```

Comment:
⛏️ Add docs and could we maintain consistency with other places where we are using `isPreAuth` instead of `preAuth`?